tox.ini

```diff
@@ -95,6 +96,8 @@ deps =
     aiobotocore: asynctest
     aiobotocore: moto>=1.0.1
     aiobotocore: flask
+    aiopg: aiopg>=0.12.0
```

Comment:
`aiohttp` should be already ok, but it's not updated with latest versions (so we're missing the 2.0, 2.1 and 2.2). We can think about them later. Same for `aiobotocore` that should follow the same approach so we're sure it works for specific versions.